Update snowflake-configs.md (#7519)
fishtown-analytics/docs.getdbt.com: website/docs/reference/resource-configs/snowflake-configs.md

<Tabs
  defaultValue="dbt_project.yml"
  values={[
    { label: 'Project file', value: 'dbt_project.yml', },

>     { label: 'Property file', value: 'models/_models.yml', },

Author: ```suggestion
    { label: 'Property file', value: 'models/my_model.yml', },
```

...

models:

>   +snowflake_warehouse: "EXTRA_SMALL"    # default snowflake virtual warehouse for all models in the project.

Reviewer: I think branding should override any historical casing we may do (but also why don't we sentence case these?)
```suggestion
  +snowflake_warehouse: "EXTRA_SMALL"    # default Snowflake virtual warehouse for all models in the project.
```

>   +snowflake_warehouse: "EXTRA_SMALL"    # default snowflake virtual warehouse for all models in the project.

Reviewer: I think branding should override any historical casing we may do (but also why don't we sentence case these?)
```suggestion
  +snowflake_warehouse: "EXTRA_SMALL"    # default Snowflake virtual warehouse for all models in the project.
```

  my_project:
    clickstream:

>       +snowflake_warehouse: "EXTRA_LARGE"    # override the default snowflake virtual warehouse for all models under the `clickstream` directory.

Reviewer: ```suggestion
      +snowflake_warehouse: "EXTRA_LARGE"    # override the default Snowflake virtual warehouse for all models under the `clickstream` directory.
```

snapshots:
  +snowflake_warehouse: "EXTRA_LARGE"    # all Snapshot models are configured to use the `EXTRA_LARGE` warehouse.
```

</File>
</TabItem>

> <TabItem value="models/_models.yml">

Author: ```suggestion
<TabItem value="models/my_model.yml">
```

> The following example changes the warehouse for a single model with a config argument in the YAML.

Author: ```suggestion
The following example overrides the Snowflake warehouse for a single model using a config argument in the property file.
```

> <File name='models/_models.yml'>

Author: ```suggestion
<File name='models/my_model.yml'>
```

```yaml
models:
  - name: my_model
    config:

>       snowflake_warehouse: "EXTRA_LARGE"    # override the snowflake virtual warehouse for just this model

Author: ```suggestion
      snowflake_warehouse: "EXTRA_LARGE"    # override the snowflake virtual warehouse just for this model
```

>       snowflake_warehouse: "EXTRA_LARGE"    # override the snowflake virtual warehouse just for this model

Reviewer: ```suggestion
      snowflake_warehouse: "EXTRA_LARGE"    # override the Snowflake virtual warehouse just for this model
```

```

</File>
</TabItem>
<TabItem value="models/events/sessions.sql">

The following example changes the warehouse for a single model with a config() block in the SQL model.

<File name='models/events/sessions.sql'>

```sql

> -- override the snowflake virtual warehouse for just this model

Reviewer: ```suggestion
-- override the Snowflake virtual warehouse for just this model
```

> -- override the Snowflake virtual warehouse for just this model

Author: ```suggestion
# override the Snowflake virtual warehouse for just this model
```
